src/index.js

```diff
@@ -2,4 +2,13 @@ import '@fortawesome/fontawesome-free/js/fontawesome';
 import '@fortawesome/fontawesome-free/js/solid';
 import '@fortawesome/fontawesome-free/js/regular';
 import './style.css';
-import './crudop.js';
\ No newline at end of file
+import taskList from './crudop.js';
+import { markCompleted, clearAll } from './clear';
+
+taskList.addEventListener('click', markCompleted);
```

Comment:
Thank you for your suggestion @Deepakdanger but when I change it to "change" this won't work anymore.
This solution is working perfectly since once you click on complete it changes the property complete to true or false, you can check in the local storage.
Thank you for understanding